Code/Mantid/Framework/API/inc/MantidAPI/NumericAxis.h

```diff
@@ -65,6 +65,8 @@ class MANTID_API_DLL NumericAxis : public Axis {
   virtual void setValue(const std::size_t &index, const double &value);
   size_t indexOfValue(const double value) const;
   virtual bool operator==(const Axis &) const;
+  virtual bool equalWithinTolerance(const Axis &axis2,
+                                    const double tolerance = 0.0) const;
```

Comment:
May we settle on a single default tolerance or remove the default value? 1e-15 is being used for operator==.